packages/templates/apollo-angular/src/helpers/ngmodule-directive.ts

```diff
@@ -0,0 +1,35 @@
+const R_DEF = /\s\@NgModule\([^)]+\)/gm; // matches: @NgModule(...)
+const R_MOD = /module\:\s*"([^"]+)"/; // matches: module: "..."
+
+// checks if operation has NgModule directive
+export function operationHasNgModule(operation) {
```

Comment:
I'm not sure that looking for it that way and then removing it from the string is the best way to deal with it.
I was thinking about letting templates the option to "inject" types/directives into the schema, and this way templates that has directives will provide it, and the codegen will add it to the schema.
This way you'll be able to check the existence of the NgModule directive using GraphQL AST. 

What do you think? @kamilkisiela